styleguide.css

```diff
@@ -283,10 +289,11 @@ form#signin {
 
 [type=checkbox]:not(:checked) + label,
 [type=checkbox]:checked + label {
+    display: block;
     position: relative;
-    padding-left: 35px;
+    padding-left: 3.5rem;
     cursor: pointer;
```

Comment:
With a `pointer` cursor on labels, won't it pretend like it's a clickable element (button or link)?

Reply:
it's on label[type=submit] only

Reply:
Right, on a label that is attached to a checkbox only 👍 

HTML also permits you to write a checkbox inside of a label:

```
<label><input type=checkbox> Accept this?</label>
```

Reply:
ah oui, label which is after a checkbox.
ah ok, didn't know. Well if the `label` is after the `input` it's fine